docs/reference/docs/reindex.asciidoc

```diff
@@ -489,6 +489,9 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=wait_for_active_shards
 (Optional, Boolean) If `true`, the request blocks until the operation is complete.
 Defaults to `true`.
 
+`ignore_unavailable`::
+(Optional, Boolean) If false, the request returns an error if it targets a missing or closed index. Defaults to false.
```

Comment:
Is this required to fix the issue? It's not a bad thing to add for sure but I'd prefer it to be a separate PR unless it's intimately linked to the bugfix.

Reply:
I added it since cross-cluster ops piggy-back regular search API. It could be very useful in [Reindex from multiple sources](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-reindex.html#docs-reindex-from-multiple-sources) use case should an user wants to continue script execution opposing to current fail-fast behaviour. I I see this APIs alignment (with search) as added value.

But, OK I'm reducing scope of this PR